tools/sggolangcilint/tools.go

```diff
@@ -53,7 +53,12 @@ func Run(ctx context.Context, args ...string) error {
 		if err != nil {
 			return err
 		}
-		cmd := Command(ctx, append([]string{"run", "-c", configPath, "--path-prefix", pathPrefix}, args...)...)
+		var excludeArg []string
```

Comment:
@Edholm made a blunder in my first commit, appending to the `args` slice made it so it was applied for all runs of the command, not just the one in the sage folder :) Please have a look again